src/serde_obkv/value/mod.rs

```diff
@@ -245,7 +246,7 @@ impl ObjMeta {
     }
 
     pub fn new_numeric_meta(obj_type: ObjType) -> ObjMeta {
-        ObjMeta::new(obj_type, CollationLevel::Numeric, CollationType::Binary, -1)
+        ObjMeta::new(obj_type, CollationLevel::Numeric, CollationType::Binary, 10)
```

Comment:
Is it safe to change the -1 to 10? Or it's a bug to use -1?

Reply:
I made a mistake here, it should be -1.
https://github.com/oceanbase/obkv-table-client-java/issues/30